packages/edit-site/src/components/page-templates/index.js

```diff
@@ -176,7 +177,7 @@ function Preview( { item, viewType } ) {
 	// the block editor settings are needed in context where we don't have the block editor.
 	// Explore how we can solve this in a better way.
 	return (
-		<ExperimentalBlockEditorProvider settings={ settings }>
+		<EditorProvider post={ item } settings={ settings }>
```

Comment:
This Preview component should actually be moved to the "editor" package and be named something like `PostPreview` or something. The purpose of that is to preview a given postType + postId tuple.

Reply:
You mean by having a lighter component than the actual EditorProvider, right? Do you think this change will affect performance for now, until we implement that component?